### PR TITLE
enchannel: Add missing dependency

### PR DESCRIPTION
**Pull request by lgeiger**
- Fixes launch
- Fixes precommit hook
- Partially fixes tests

### CODE REVIEW

File: packages/desktop/package.json [hunk 1]
```diff
@@ -101,9 +100,6 @@
   "jest": {
     "setupFiles": ["./scripts/mockument"]
   },
-  "lint-staged": {
-    "*.js": ["prettier --write", "git add"]
-  },
```
Comment (rgbkrk) [inline packages/desktop/package.json:106]: We still want lint staged at the top right?

**Review by rgbkrk — APPROVED** [review]

**Comment by rgbkrk** [comment]
Feel free to add on to the PR directly.

**Comment by rgbkrk** [comment]
Whoops, I hit send too fast.

Thanks for hacking on this. I'm totally cool with you contributing directly to the mega monorepo branch by the way.

File: packages/desktop/package.json (hunk 1 above)
Comment (lgeiger) [reply packages/desktop/package.json:106]: Yes we do

**Pull request merged**